### DISCUSSION TITLE
new bug after update

### DISCUSSION THREAD

**Discussion by Le0-o** [siweia/NDui#311] [Q&A]
Message: Interface/AddOns/NDui/Modules/Infobar/Guild.lua:319: attempt to concatenate local 'online' (a nil value)
Time: Wed Nov 20 19:31:30 2024
Count: 1
Stack: Interface/AddOns/NDui/Modules/Infobar/Guild.lua:319: attempt to concatenate local 'online' (a nil value)
[string "=[C]"]: ?
[string "@Interface/AddOns/NDui/Modules/Infobar/Guild.lua"]:319: in function onEvent'
[string "@Interface/AddOns/NDui/Modules/Infobar/Core.lua"]:155: in function Infobar_UpdateAnchor'
[string "@Interface/AddOns/NDui/Modules/Infobar/Core.lua"]:193: in function OnLogin'
[string "@Interface/AddOns/NDui/Init.lua"]:125: in function func'
[string "@Interface/AddOns/NDui/Init.lua"]:27: in function <Interface/AddOns/NDui/Init.lua:22>

and

Message: Interface/AddOns/NDui/Modules/Infobar/Guild.lua:319: attempt to concatenate local 'online' (a nil value)
Time: Wed Nov 20 19:37:57 2024
Count: 61
Stack: Interface/AddOns/NDui/Modules/Infobar/Guild.lua:319: attempt to concatenate local 'online' (a nil value)
[string "=[C]"]: ?
[string "@Interface/AddOns/NDui/Modules/Infobar/Guild.lua"]:319: in function onEvent'
[string "@Interface/AddOns/NDui/Modules/Infobar/Core.lua"]:62: in function <Interface/AddOns/NDui/Modules/Infobar/Core.lua:60>

**Comment by siweia** [comment]
keep your addon up to date

**Comment by Le0-o** [comment]
Already did but last version is before server update time, issue still exists


﻿

keep your addon up to date
